docs/architecture/baking.md

```diff
@@ -27,25 +27,62 @@ It packages pending operations into a block, signs it, and distributes it to the
 
 ## Becoming a baker
 
-To become a baker, you must create an account to act as a _delegate_, which is an account that is authorized to bake blocks and attest blocks that other accounts bake.
+To become a baker, you must create an account to act as a _delegate_, which is an account that is authorized to bake blocks and attest blocks that other accounts bake, and also to receive delegations from other accounts (as explained later).
 Delegates temporarily stake tez as a security deposit to ensure that they are acting according to the rules of the protocol.
-This stake stays locked for a certain number of cycles before the delegate can begin baking.
+There is a delay of a certain number of cycles from the moment when the tez are staked until the delegate can begin baking.
 Similarly, when bakers stop baking, their stake is unlocked after a certain number of cycles.
+While it is locked, the security deposit may be slashed by the protocol if the baker misbehaves (e.g., proposes or attests two different blocks for the same level).
 
-The delegate must stake at least least 6,000 tez, either from its own account or from tez that delegators delegate to it.
-The more tez a delegate has, both from its own account and from delegators, the more likely it is to be selected to bake a block and thus receive the rewards.
+A delegate participates in consensus in proportion to their _baking power_: the more baking power a delegate has, the more likely it is to be selected to bake or to validate blocks and thus receive the rewards.
+The baking power of a delegate is computed from the amounts of tez owned by its own and by all the accounts that delegate to it, knowing that non-staked tez are weighted half as much as staked tez in the sum.
```

Comment:
The baking power is computed by the staked balance of the baker (from its own stake, plus that of their stakers)  and the delegated balance (their own, their stakers and delegators).

Delegated tez give rights to half their nominal tez value.

Reply:
NB: We should flag already in some issue what might need to be updated for Quebec (that is, that the baking formula changes)

Reply:
Thanks, I rewrote along your suggestions, but keeping it a bit more accessible.